code/game/machinery/vending_types.dm

```diff
@@ -345,23 +345,23 @@
 		/obj/item/storage/fancy/cigar = 5
 	)
 	prices = list(
-		/obj/item/storage/fancy/cigarettes/rugged = 200,
-		/obj/item/storage/fancy/cigarettes = 245,
-		/obj/item/storage/fancy/cigarettes/dromedaryco = 270,
-		/obj/item/storage/fancy/cigarettes/nicotine = 325,
-		/obj/item/storage/chewables/rollable/bad = 40,
-		/obj/item/storage/chewables/rollable = 50,
-		/obj/item/storage/chewables/rollable/fine = 60,
-		/obj/item/storage/chewables/rollable/nico = 90,
-		/obj/item/storage/chewables/tobacco/bad = 60,
-		/obj/item/storage/chewables/tobacco = 90,
-		/obj/item/storage/chewables/tobacco/fine = 120,
-		/obj/item/storage/fancy/chewables/tobacco/nico = 150,
+		/obj/item/storage/fancy/cigarettes/rugged = 10,
+		/obj/item/storage/fancy/cigarettes = 15,
+		/obj/item/storage/fancy/cigarettes/dromedaryco = 25,
+		/obj/item/storage/fancy/cigarettes/nicotine = 30,
+		/obj/item/storage/chewables/rollable/bad = 5,
+		/obj/item/storage/chewables/rollable = 10,
+		/obj/item/storage/chewables/rollable/fine = 15,
+		/obj/item/storage/chewables/rollable/nico = 20,
+		/obj/item/storage/chewables/tobacco/bad = 20,
+		/obj/item/storage/chewables/tobacco = 25,
+		/obj/item/storage/chewables/tobacco/fine = 40,
+		/obj/item/storage/fancy/chewables/tobacco/nico = 15,
```

Comment:
The reason why it's so expensive is because it's literally the **pure** nicotine reagent inside.